api.go

```diff
@@ -645,10 +645,12 @@ func StatusContainer(podID, containerID string) (ContainerStatus, error) {
 				PID:    container.process.Pid,
 				RootFs: container.config.RootFs,
 			}
+			return contStatus, nil
```

Comment:
Would be more idiomatic to get rid of the contSatus variable altogether with:

````
			return ContainerStatus{
				ID:     container.id,
				State:  container.state,
				PID:    container.process.Pid,
				RootFs: container.config.RootFs,
			}, nil
````

Wondering if returning a pointer instead of coping the struct would be a good idea as well, but that's another discussion though.